test/fixtures/clients.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  email: MyString
+  phone: MyString
+
+two:
+  email: MyString
+  phone: MyString
```

Comment:
This fixture won't work accordingly to https://github.com/rakvium/taxi-api/pull/15/files#diff-0a8fa39aebf1877d55271c814b471dadR10